pkg/volume/csi/csi_plugin_test.go

```diff
@@ -1514,3 +1522,55 @@ func TestHighestSupportedVersion(t *testing.T) {
 		}
 	}
 }
+
+func TestPluginConversionToPluginHandler(t *testing.T) {
+	testCases := map[string]struct {
+		genericPlugin  volume.VolumePlugin
+		expectedErrMsg string
+	}{
+		"nil cannot be type-casted to csiPlugin": {
+			genericPlugin:  nil,
+			expectedErrMsg: "is not a CSI volume plugin",
```

Comment:
Would test be easier if you simply check for existence of err instead of comparing error string ? A simple string change in error string (case change, error correction, etc) will cause test to fail.

Reply:
Yeah, I had different error cases in mind somewhen. But that's not the case anyway, so absolutely, we should just check on error or not here.